privval/utils.go

```diff
@@ -18,3 +23,39 @@ func IsConnTimeout(err error) bool {
 	}
 	return false
 }
+
+// NewSignerListener creates a new SignerListenerEndpoint using the corresponding listen address
+func NewSignerListener(listenAddr string, logger log.Logger) (*SignerListenerEndpoint, error) {
+	var listener net.Listener
+
+	protocol, address := cmn.ProtocolAndAddress(listenAddr)
+	ln, err := net.Listen(protocol, address)
+	if err != nil {
+		return nil, err
+	}
+	switch protocol {
+	case "unix":
+		listener = NewUnixListener(ln)
+	case "tcp":
+		// TODO: persist this key so external signer can actually authenticate us
```

Comment:
link to the issue?

Reply:
this is actually code that was moved. I am not sure what was the meaning or intention behind that comment :(

Reply:
Okay